pkgconfig/README.md

```diff
@@ -52,8 +52,9 @@ $ clang++-11 ${cxxflags} -o enc enc.cpp ${libs}
 To build an enclave application with a specific crypto library, additional commands are needed to incorporate additional source files and headers. In this example, `OE_CRYPTO_LIB` is set to `openssl`. The values that `OE_CRYPTO_LIB` supports are:
 - `mbedtls`
 - `openssl`
-- `openssl_symcrypt_fips`
+- `symcrypt_fips`
```

Comment:
This is a breaking change. No one might be using openssl_symcrypt_fips, but nevertheless, it might be better to preserbe it.

Reply:
flexible to change my mind, if @mingweishih  chimes in otherwise.

Reply:
I agree. let's avoid this breaking change

Reply:
Resolved